config/helm/crd_kubebuilder_version.yaml

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
I think we added this on purpose to avoid changing many files when only bumping the controller-gen version was needed.

Reply:
https://github.com/giantswarm/cluster-api-provider-azure-app/pull/72

Reply:
We have azure-service-operator in 1.12 and its kubebuilder version is different. We have CRDs whose versions are different now.